package.json

```diff
@@ -14,13 +14,17 @@
   },
   "homepage": "https://github.com/FormidableLabs/formidable-react-component-boilerplate",
   "scripts": {
-    "prepublish": "npm run build",
+    "postinstall": "npm run build-lib",
+    "preversion": "npm run check",
+    "version": "npm run clean && npm run build && git add -A dist",
+    "postversion": "git push && git push --tags",
```

Comment:
This `postversion` hook feels a little bit scary to me. What if you accidentally `npm version major` when you meant to `minor`, for example? Easy to fix before it's pushed to GH, harder/more annoying after.